### DISCUSSION TITLE
Streamline Visualizations

### DISCUSSION THREAD

**Discussion by jab0707** [SCIInstitute/SCIRun#2210] [Q&A]
This may be a bug report or feature request but I have a vector field I want to visualize with streamlines. The problem is when I use showfieldglyphs it default selects points as the display type. This makes for very thin streamlines that are hard to actually see when there are a lot of them. I have tried switching the option to sphere, but that 1) takes up way too much RAM and instantly freezes the computer, and 2) generates spheres that are way too huge, as int he block out everything int he viewscene, even if scale is set to 0.0001. Is there a better way to visualize thicker streamlines?

**Comment by dcwhite** [comment]
This sounds like a job for Ospray. @jessdtate do you agree?

**Reply by jessdtate** [reply]
ospray has some optimized stream line and spheres methods, so it would probably help.

**Comment by dcwhite** [comment]
Also some example screenshots would be nice

**Comment by dcwhite** [comment]
One quick fix might be to allow the scale double spinboxes to have a wider range, so that you can scale your data down even more.

**Reply by jessdtate** [reply]
for the sphere sizes.  if you try normalizing your data before scaling, you can get an idea of the size of glyphs that you might want.  another quick fix could also be to use CalculateFieldData to adjust the data range and use it in the secondary port to change the glyph size.

**Comment by jessdtate** [comment]
I'm not sure why you are using showfield glyphs for streamlines.  it would probably be more efficient to use showfield with the curvemesh (edges turned on).
